src/sentry/web/frontend/error_page_embed.py

```diff
@@ -92,41 +92,44 @@ def _get_project_key(self, request):
     def _get_origin(self, request):
         return origin_from_request(request)
 
-    def _json_response(self, request, context=None, status=200):
-        if context:
-            content = json.dumps(context)
-        else:
+    def _smart_response(self, request, context=None, status=200):
+        json_context = json.dumps(context or {})
+        accept = request.META.get('HTTP_ACCEPT') or ''
+        if 'text/javascript' in accept:
+            content_type = 'text/javascript'
             content = ''
-        response = HttpResponse(content, status=status, content_type='application/json')
+        else:
```

Comment:
i chose to blindly respond with JSON as a default since theres no real other use case